platform-controller/src/main/java/org/hobbit/controller/docker/DockerUtility.java

```diff
@@ -0,0 +1,27 @@
+package org.hobbit.controller.docker;
+
+import com.spotify.docker.client.DefaultDockerClient;
+import com.spotify.docker.client.DockerClient;
+import com.spotify.docker.client.exceptions.DockerCertificateException;
+
+public class DockerUtility {
+    private static DockerClient dockerClient = null;
+
+    protected DockerUtility() {
+        // Exists only to defeat instantiation.
+    }
+
+    public static DockerClient getDockerClient() throws DockerCertificateException {
```

Comment:
If you want to make the dockerClient a singleton, you need to make this method `synchronized`.